### PR TITLE
[WIP] Add python bindings for `cudf::fill` and `cudf::copy_range`

### PR DESCRIPTION
**Pull request by RFinkelberg**
This PR contributes python bindings for the libcudf functionality introduced by #1908 

(dependent on #1908)

### CODE REVIEW

File: python/cudf/bindings/copying.pyx [hunk 1]
```diff
@@ -234,3 +235,30 @@ def apply_scatter_array(dev_array, maps, out_col=None):
...
+        column to copy from
+    idf_out_begin, idx_out_end : gdf_index_type
+        [start, end) indices in out_col to copy to
+    idx_in_begin : gdf_index_type
+        index in in_col to start copying from
+    """
+    check_gdf_compatibility(out_col)
+    check_gdf_compatibility(in_col)
+    cdef gdf_column *c_out_col = column_view_from_column(out_col)
+    cdef gdf_column *c_in_col = column_view_from_column(in_col)
```
Comment (kkraus14) [inline python/cudf/bindings/copying.pyx]: You're leaking both of these, you should free them after the `copy_range(...)` call below.

File: python/cudf/bindings/filling.pyx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ---------
+    column : Column
+        Column to fill
+    value : scalar
+        Scalar value to fill column with
+    idx_begin, idx_end : gdf_index_type
+        Beginning and end indices to fill between
+    """
+    check_gdf_compatibility(column)
+    cdef gdf_column *out_col = column_view_from_column(column)
```
Comment (kkraus14) [inline python/cudf/bindings/filling.pyx]: You're leaking this, should be freed after the `fill(...)` call below.

**Comment by harrism** [comment]
Are there plans to plumb this into the cudf module in a way that users can access (e.g. via `__setitem__`)?

**Comment by kkraus14** [comment]
> Are there plans to plumb this into the cudf module in a way that users can access (e.g. via `__setitem__`)?

Exactly 😄

**Comment by harrism** [comment]
@RFinkelberg Please update the changelog to pass the CI changelog check.

**Comment by shwina** [comment]
Closing as this is superseded by #2442

**Pull request closed without merging**